tests/operators/plusequal_test.cpp

```diff
@@ -0,0 +1,31 @@
+#include <iostream>
+#include <cassert>
+
+#include "../../src/bigint.hpp"
+
+int main() {
+  freopen("input.txt", "r", stdin);
```

Comment:
Codacy found an issue: [Return value of function freopen() is not used.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285288)